SA3/qa_web/templates/qa_web/answerspage.html

```diff
@@ -60,12 +60,7 @@ <h6>Question by {{ currentQuestion.owner }} on {{ currentQuestion.creation_date
         </p>
```

Comment:
Line 58 in this file: `<span id='score_{{ comment.id }}_comment'>{{ comment.score }}</span>`

This is not your fault but since the PR for comments for questions, we forgot to add a `class = 'marked' ` for the span. Without this, comments for questions does not display as MD. 

Can you quickly make a commit for it, as its implementation is parallel to the topic of this PR?